Userland/Libraries/LibWeb/XHR/XMLHttpRequest.h

```diff
@@ -63,8 +63,11 @@ class XMLHttpRequest final : public XMLHttpRequestEventTarget {
 
     WebIDL::ExceptionOr<void> override_mime_type(String const& mime);
 
-    WebIDL::ExceptionOr<void> set_timeout(u32 timeout);
     u32 timeout() const;
+    WebIDL::ExceptionOr<void> set_timeout(u32 timeout);
```

Comment:
Should moving this be a separate commit?
(I don't think it's needed but you do have a commit fixing some stuff in this file already)

Reply:
Don't think that's worth its own commit tbh, it's only to keep them consistent when adding the other getter/setter pair.